### PR TITLE
change screensaver to boucing shiftcrypto logo

### PR DESCRIPTION
**Pull request by My1**

### CODE REVIEW

File: src/ui/components/screensaver.c [hunk 1]
```diff
@@ -30,20 +30,36 @@ static void _render(component_t* component)
     static uint16_t counter = 0;
     counter++;
 
-    const uint16_t slowdown_factor = 6; // slow it down a bunch
+    // these flip between positive and negative when boncing, can also be used to move multiple
+    // pixels per frame
+    static int8_t x_direction = 1;
+    static int8_t y_direction = 1;
+    // setting relative speed for both axes
+    static int8_t x_slowdown = 1;
```
Comment (benma) [inline src/ui/components/screensaver.c]: those can be `const` instead of `static`.

**Review by benma — CHANGES_REQUESTED** [review]
Works flawlessly, great work!

**Comment by My1** [comment]
> Works flawlessly, great work!

was a ton of "fun" yesterday to get it to not escape, especially now that I have seen the screensaver reset, no wonder the bounce logic screwed me so often

File: src/ui/components/screensaver.c (hunk 1 above)
Comment (My1) [reply src/ui/components/screensaver.c]: well the slowdowns yes but the directions would probably not really work well as const as they are flipped all the time

**Comment by My1** [comment]
for reference the image I used

bitbox-screensaver

**Comment by benma** [comment]
:clap:

**Pull request merged**